src/main/java/com/amazon/ion/impl/bin/WriteBuffer.java

```diff
@@ -21,7 +21,7 @@
  * in an IndexOutOfBoundsException. The number 10 is chosen because it is the maximum number of bytes required to write
  * a long value as a FlexInt or VarInt.
  */
-/*package*/ final class WriteBuffer implements Closeable
+public final class WriteBuffer implements Closeable
```

Comment:
This is being used in Macroize, which is in a different Java package. We can rename this class using the `_Private_` prefix if we want to maintain the existing convention to denote that this shouldn't be used by users of the library.